src/client/kotlin/me/owdding/skyocean/features/item/search/screen/ItemSearchScreen.kt

```diff
@@ -235,16 +288,46 @@ object ItemSearchScreen : SkyOceanScreen() {
                     else -> {}
                 }
                 context.collectLines().forEach(::add)
-            }.withPadding(2).asButton { button ->
+            }.withPadding(2)
+
+
+            val leftAction = { button: Button ->
                 ItemHighlighter.setHighlight(ReferenceItemFilter(itemStack))
                 context.open()
                 McClient.setScreen(null)
             }
+
+            if (context is SackItemContext) {
```

Comment:
Could add, that on bundle items it also allows rightclick if there is at least on sack item present